x-pack/test/api_integration/deployment_agnostic/apis/observability/dataset_quality/degraded_fields.ts

```diff
@@ -178,20 +178,15 @@ export default function ({ getService }: DeploymentAgnosticFtrProviderContext) {
           (dFields: DegradedField) => dFields.name === 'trace.id'
         )?.indexFieldWasLastPresentIn;
 
-        expect(logLevelLastBackingIndex).to.be(
-          `${createBackingIndexNameWithoutVersion({
```

Comment:
AFAIK we are not using that helper in any other place `createBackingIndexNameWithoutVersion`, can we get rid of it?